test_data/pro/stac_responses/items_page_1_limit_500.json

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
are we allowed to store this file in our repo license wise?

Reply:
Hm, the STAC API is under MIT https://github.com/stac-utils/stac-server
Maybe we need to remove the URLs?
But there is not much Schöpfungshöhe.